ramls/actionResponse.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Can we split this in two separate responses - one for success and one for failure?